docs/Selenium.md

```diff
@@ -0,0 +1,13 @@
+## Selenium
+
+### Prerequisites
+- [Python](https://www.python.org/downloads/)
+- pytest  `pip install pytest`
+- selenium  `pip install selenium`
+- webdriver-manager  `pip install webdriver-manager`
```

Comment:
Installing packages should be with a specific version so if a higher version is released and broken, we can go back to a working version. we can do that with poetry with `pyproject.toml`, or at least add them in a `requirements.txt`